packages/web/src/common/store/pages/signon/types.ts

```diff
@@ -49,6 +49,7 @@ export default interface SignOnPageState {
   otp: EditableField
   handle: EditableField
   genres: string[]
+  // TODO-NOW: remove, this is added during social account flow
```

Comment:
Still need TODO?

Reply:
Yes. I have a handful of TODOs to clean up. Thanks for calling it out.